pytket/extensions/cutensornet/utils.py

```diff
@@ -0,0 +1,92 @@
+from numpy.typing import NDArray
+from pytket.backends.backendresult import BackendResult
+from pytket.circuit import Qubit, Circuit  # type: ignore
+
+
+def _reorder_qlist(
+    post_select_dict: dict, qlist: list[Qubit]
+) -> tuple[list[Qubit], Qubit]:
+    """Reorder qlist so that post_select_qubit is first in the list.
+
+    Args:
+        post_select_dict (dict): Dictionary of post selection qubit and value
+        qlist (list): List of qubits
+
+    Returns:
+        tuple: Tuple containing: q_list_reordered (list): List of qubits
+        reordered so that post_select_qubit is first in
+        the list. q (Qubit): The post select qubit
+    """
+
+    post_select_q = list(post_select_dict.keys())[0]
```

Comment:
Seems like you could just pass the first key rather than a dict. Also, I think this function can be reduced to:
```
qlist[qlist.index(post_select_qubit)], qlist[0] = qlist[0], qlist[qlist.index(post_select_qubit)]
```

Reply:
Actually, I realised this will re-order remaining qubits, so probably not a good idea (if it matters)?